index.js

```diff
@@ -1,3 +1,4 @@
 export { parse } from "./lib/webidl2.js";
 export { write } from "./lib/writer.js";
 export { validate } from "./lib/validator.js";
+export {WebIDLParseError} from "./lib/tokeniser.js";
```

Comment:
```suggestion
export { WebIDLParseError } from "./lib/tokeniser.js";
```

For code format consistency 😁 (You can click "Commit suggestion" to automatically accept my suggestion)